internal/pipe/docker/api.go

```diff
@@ -69,7 +69,7 @@ func runCommandWithOutput(ctx *context.Context, dir, binary string, args ...stri
 	/* #nosec */
 	cmd := exec.CommandContext(ctx, binary, args...)
 	cmd.Dir = dir
-	cmd.Env = ctx.Env.Strings()
+	cmd.Env = append(cmd.Env, ctx.Env.Strings()...)
```

Comment:
I think this should be cmd.Environ(), no?

Reply:
https://github.com/goreleaser/goreleaser/commit/cbb09b89fd3297129aaba89f57240c32bda6aa41

Reply:
Yep, good catch. Thx for fixing